<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leader</class>
 <widget class="QWidget" name="Leader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>695</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Leader Window Style */
QMainWindow {
    background-color: #f5f5f5; /* Светло-серый фон для главного окна */
    border: 1px solid #dcdcdc; /* Граница окна */
}

/* Tab Widget Style */
QTabWidget {
    border: 2px solid #dcdcdc; /* Граница для таблицы */
    border-radius: 5px; /* Скругленные углы */
    background-color: #ffffff; /* Белый фон */
    font-size: 14px; /* Размер шрифта */
}

/* Table Header Style */
QHeaderView::section {
    background-color: #0078d7; /* Синий фон заголовка таблицы */
    color: #ffffff; /* Белый цвет текста */
    padding: 8px; /* Отступ внутри заголовка */
}

/* Line Edits */
QLineEdit {
    border: 2px solid #dcdcdc; /* Граница для полей ввода */
    border-radius: 5px; /* Скругленные углы */
    padding: 5px; /* Отступ внутри поля ввода */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
}

QLineEdit:focus {
    border: 2px solid #0078d7; /* Цвет границы при фокусе */
}

/* Buttons */
QPushButton {
    background-color: #0078d7; /* Синий фон кнопки */
    color: #ffffff; /* Белый цвет текста */
    border: none; /* Убираем границу */
    border-radius: 5px; /* Скругленные углы */
    padding: 10px; /* Отступ внутри кнопки */
    font-size: 14px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: #005bb5; /* Темно-синий фон при наведении */
}

QPushButton:pressed {
    background-color: #004494; /* Еще более темный фон при нажатии */
}

/* Labels */
QLabel {
    font-size: 14px; /* Размер шрифта */
    color: #333333; /* Темно-серый цвет текста */
}

/* Radio Buttons */
QRadioButton {
    font-size: 14px; /* Размер шрифта */
}

/* Message Boxes */
QMessageBox {
    background-color: #ffffff; /* Белый фон для окон сообщений */
    border: 1px solid #dcdcdc; /* Граница окна сообщения */
}
</string>
  </property>
  <widget class="QTabWidget" name="LeaderTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>650</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Cписок пользователей</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="UserTableWidget">
       <column>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Отчество</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Должность</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Степень/звание</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата добавления</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Список выполняющихся работ</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidget_2">
       <column>
        <property name="text">
         <string>Наименование</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Отчество</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Сложность</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Трудоёмкость</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Оплата</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Подтвердить выполненную работу</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <underline>true</underline>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Сотрудники, которые выполнили работу:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Выберите одного или несколько сотрудников</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_3">
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Отчество</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Наименование</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Комментарий:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CommentLine"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>Сотрудник выполнил работу?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>Выберите один из вариантов:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="RadioButtonYes">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RadioButtonNo">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Нет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SendCompleteButton">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Добавить дополнительную работу</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Введите наименование работы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameWorkLine">
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>Оплата:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="PaymentLine">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>рублей                                                                                                                                                   </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Введите сложность и трудоёмкость работы:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Сложность:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="HardSpinBox">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Трудоёмкость:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSpinBox" name="ComplexitySpinBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>Выберите одного или нескольких сотрудников:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_4">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Фамилия</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Отчество</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Комменатрий:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CommentLeaderLine">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SendLeaderButton">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>891</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Выйти из системы</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
